src/convert/from_cellranger_multi_to_h5mu/config.vsh.yaml

```diff
@@ -0,0 +1,49 @@
+functionality:
+  name: "from_cellranger_multi_to_h5mu"
+  namespace: "convert"
```

Comment:
I would consider calling it `mapping/cellranger_multi_to_h5mu` so that it's closer to the component it's trying to convert from. Does that make sense?

Reply:
I just followed the convention from the other components. Perhaps we should rename those as well?
<img width="292" alt="Screenshot 2022-11-17 at 13 11 12" src="https://user-images.githubusercontent.com/5946712/202443051-46714ace-2d27-493f-b18e-55f098f28cee.png">

Reply:
I realised ^^

We can work on deprecating those components, yes. We could even:

Contents of `src/convert/from_bdrhap_to_h5mu/config.vsh.yaml`:
```yaml
__inherits__: ../../mapping/bd_rhapsody_to_h5mu/config.vsh.yaml
functionality:
  name: from_bdrhap_to_h5mu
  namespace: convert
  status: deprecated
```

:grimacing:

Reply:
I would propose to make that a seperate PR?

Reply:
Ok!